projects/robots/robotis/darwin-op/libraries/Makefile

```
@@ -54,10 +54,14 @@ endif
 	+PYTHON_COMMAND=python3.7 make -s -C python $(MAKECMDGOALS)
 	@echo "# make" $@ python3.8;
 	+PYTHON_COMMAND=python3.8 make -s -C python $(MAKECMDGOALS)
+	@echo "# make" $@ python3.9;
+	+PYTHON_COMMAND=python3.9 make -s -C python $(MAKECMDGOALS)
 endif
 ifeq ($(OSTYPE),windows)
+	@echo "# make" $@ python3.9;
+	+PATH="$(PYTHON39_HOME):$(PATH)" make -s -C python $(MAKECMDGOALS)
```

Can't we simply have this?
```suggestion
	+make -s -C python $(MAKECMDGOALS)
```
As `PYTHON39_HOME` should already be in the `PATH`, isn't it?

---

Yes, sure we can as in theory `scripts/install/msys.sh` is already setting it.
But I thought it was more homogeneous like this (less special cases means less sources of errors) and more handy/safer (I already encountered several times on Windows the case where I changed manually my PATH to point to python 2.7 and then the compilation failed because of this).
But I don't have any strong opinion on this, so if you prefer that I remove it it's fine too for me.

---

I see. It's true, that it is more robust like that, and it is often useful to change the PATH environment variable to make some tests, so let's keep it like this.